src/Morphir/Visual/ViewApply.elm

```diff
@@ -42,12 +51,36 @@ view config viewDefinitionBody viewValue functionValue argValues =
                 , openElement = openElement
                 , openHeader = openHeader
                 , isOpen = isOpen
+                , zIndex = zIndex - 100
                 }
 
         viewFunctionValue : FQName -> Element msg
         viewFunctionValue fqName =
             el [ tooltip above (functionOutput fqName) ] <| viewValue functionValue
 
+        viewArgumentList : List (Element msg)
+        viewArgumentList =
+            argValues
+                |> List.indexedMap
+                    (\ind v ->
+                        case v of
+                            Value.Reference _ _ ->
+                                viewValue v
+
+                            Value.Apply _ _ _ ->
+                                viewValue v
+
+                            _ ->
+                                el
+                                    [ htmlAttribute (style "filter" "brightness(96%)")
```

Comment:
Can we simply use a background color here instead of a filter? In case of a drill-down the brightness filter serves the specific purpose of showing depth. We don't need that for arguments. A background with a low alpha would be much simpler.